ts/Series/Organization/OrganizationSeries.ts

```diff
@@ -410,6 +410,20 @@ class OrganizationSeries extends SankeySeries {
             shapeArgs.height;
     }
 
+    public drawDataLabels(): void {
+        const dlOptions = this.options.dataLabels;
+
+        if (dlOptions && dlOptions.linkTextPath &&
```

Comment:
```suggestion
        if (dlOptions.linkTextPath &&
```

DlOptions always is defined after your change in `OrganizationSeriesOptions.d.ts`